src/blog/models.py

```diff
@@ -0,0 +1,30 @@
+from constants import COMPUTING_ID_LEN
+from database import Base
+from sqlalchemy import Column, DateTime, ForeignKey, String, Text
+
+
+# blog table
+class BlogPosts(Base):
+    # table name
+    __tablename__ = "blog_posts"
+
+    # title of post ( meant to be an unique key but for simplicity,
+    # using already defined primay_key )
+    title = Column(String(128), primary_key=True, nullable=False)
+
+    # computing id
+    # TODO: add foreign key
+    computing_id = Column(String(COMPUTING_ID_LEN),
+        # ForeignKey(),

```

Comment:
Oh, just as an fyi, sqlalchemy has support for something called a `relationship` to simplify the access pattern of tables related by foreign keys. Due to restrictions of the library with async databases, we're planning on not using this feature.

Mentioning it as you might see the feature in the documentation.

Reply:
yea relationships be silly

Implementing foreign keys using equality is sorta bad practice but who cares, it's simple, maybe just do that for now 😇 

I.e., select from another table where computing_id = this.

Reply:
Do you mean bad practice with sqlalchemy? Do you know why? 

It actually kinda simplifies creating alembic migrations since alembic doesn't have support for relationships, which might confuse some people (since relationships aren't even stored in the schema).

Reply:
it's bad practice in general, for databases, to not use foreign keys when the actual use of a column is that of a foreign key. I.e. you could make an insertion that is invalid and it wouldn't throw an error

BUT if you're aware of these impacts and intend to mitigate them programmatically, who cares :))

Reply:
I'm referring to foreign keys / database constructs in general rather than alembic relationship BTW - that seems to be an oddity that i dont exactly like LMAO

Reply:
Yea I was expecting to see a ForeignKeyConstraint here